src/elements/Asset.php

```diff
@@ -2674,6 +2674,9 @@ protected function metaFieldsHtml(bool $static): string
                 'required' => true,
                 'class' => ['text', 'filename'],
                 'disabled' => $static,
+                'data' => [
+                    'attribute' => 'newLocation',
+                ],
```

Comment:
Couldn’t this have just been a top-level `attribute` config, and then no change to `Cp::fieldHtml()` needed?

Reply:
Yes, it could. Pushed an update.